src/app.js

```diff
@@ -1,32 +1,29 @@
-/* eslint-disable import/prefer-default-export */
-const todoList = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Complete Todo project',
-    completed: true,
-    index: 1,
-  },
-];
+const STORAGE_KEY = 'todos';
 
-export function displayTodoList() {
-  const todoListElement = document.getElementById('todo-list');
-  todoListElement.innerHTML = '';
-  todoList.forEach((task) => {
-    const taskElement = document.createElement('li');
-    taskElement.innerHTML = `
-        <label>
-          <input type="checkbox" class="task-checkbox" ${task.completed ? 'checked' : ''}>
-          <span class="task-text">${task.description}
+export function readTask() {
+  const todos = JSON.parse(localStorage.getItem(STORAGE_KEY)) || [];
+  return todos;
+}
 
+export function createTask(text) {
+  const todos = readTask();
+  const newTask = {
+    text,
+    completed: false,
+  };
+  todos.push(newTask);
+  localStorage.setItem(STORAGE_KEY, JSON.stringify(todos));
+  return newTask;
+}
```

Comment:
- [ ] You've done a great job💪 creating the functionality to add a new todo. However,🤷‍♂️ it seems that you're missing the 'index' property in your `newTask` object. 🙏Please make sure to add the `index` property to your object and ensure that the objects stored in local storage have an `index` property as well. When you delete❌ a todo, you should update the indexes of all the remaining todos so that your todo list represents the current list order and each item has a unique index. This will help ensure that your project has unique IDs, which is the best 👍practice when rendering a list.🙏 Please refer to the below project requirements below for more details.
-  By default, new tasks should have the property `completed` set to `false` and the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.